plugins/chat/admin/assets/javascripts/admin/components/chat-admin-plugin-actions.gjs

```diff
@@ -0,0 +1,39 @@
+import Component from "@glimmer/component";
+import { action } from "@ember/object";
+import { inject as service } from "@ember/service";
```

Comment:
```suggestion
import { service } from "@ember/service";
```

(applies to all the other places)

Reply:
My bad, I think this is still in my snippets. Will go change it